### PR TITLE
Add toString and 2x2 Matrix

### PR DESCRIPTION
**Pull request by shiraji**
Hi, I was trying to implement `inverse` method for `Mat4` (#7)  but before doing that I think creating `determinant` and `minor` method is mandatory. 

Also, I added displaying matrix method `toDisplayString` for debugging purpose. 

What do you think?

### CODE REVIEW

**Comment by shiraji** [comment]
I thought you may use `toString` method in other way. I will change them to `toString`

Sure we could limit the allocation memory but the `inverse` method become really huge. Also, `minor` can be useful as a matrix function.

**Comment by shiraji** [comment]
I think it took too much time for trying to add `inverse` method.

Could you check adding `toString()` and `Mat2` first?

File: src/main/kotlin/com/curiouscreature/kotlin/math/Matrix.kt [hunk 1]
```diff
@@ -22,6 +22,91 @@ enum class MatrixColumn {
...
+    }
+
+    fun toFloatArray() = floatArrayOf(
+            x.x, y.x,
+            x.y, y.y
+    )
+
+    override fun toString(): String {
+        return """
+            |${x.x} ${x.y}|
```
Comment (romainguy) [inline src/main/kotlin/com/curiouscreature/kotlin/math/Matrix.kt]: The String representation is incorrect. the X vector is the first column. The output should therefore be:

x.x y.x
x.y y.y

Comment (shiraji) [reply src/main/kotlin/com/curiouscreature/kotlin/math/Matrix.kt]: Fixed and squashed them

**Pull request merged**